documentation/slate/source/includes/realtime.md

```diff
@@ -0,0 +1,263 @@
+}
+```
+
+In the list of "date_times" available in the response, the parameter "data_freshness" is "realtime" and the fileld "departure_date_time" take the delay into account.
+
+A list of the disruptions impacting the stop schedules is also present at the root level of the response.
+
+## <a name="REDUCED_SERVICE"></a>Reduced service
+
+![image](reduced_service.png)
+
+``` shell
+# Extract of an impacted stop of /disruptions
+{
+    "arrival_status": "deleted",
+    "cause": "",
+    "departure_status": "deleted",
+    "is_detour": false,
+    "stop_point": ⊕{7 items},
+    "stop_time_effect": "deleted",
+}
+```
+The effect of the disruption is `REDUCED_SERVICE`. It means that the train won't be serving one or more stations in its journey.
+
+In the disruption, the deleted stations can be found in the list of "impacted_stops" with the departure/arrival status set to "deleted". See the [disruption](#disruption) objects section for its full content and description.
+
+### Journeys
+
+If the station deleted is the destination of the journey, Navitia will compute an other itinerary to the requested station. If not, the disruption will be present at the root level of the response for information, but it won't be affecting the journey.
+
+### Departures & Stop Schedules
+
+The departure time of the train with the reduced service simply won't be displayed in the list of departures/stop_schedules if "data_freshness" is set to "realtime".
+
+## <a name="MODIFIED_SERVICE"></a>Modified service
+
+![image](modified_service.png)
+
+``` shell
+# Extract of an impacted stop of /disruptions
+{
+    "amended_arrival_time": "185500",
+    "amended_departure_time": "190000",
+    "arrival_status": "added"
+    "cause": "Ajout d'une desserte",
+    "departure_status": "added",
+    "is_detour": false,
+    "stop_point": ⊕{7 items},
+    "stop_time_effect": "added",
+},
+```
+The effect of the disruption is `MODIFIED_SERVICE`. It means that there is one or several stop points added into the journey. This can be at any position in the journey (origin and destination included).
+
+In the disruption, new stop points can be found in the list of "impacted_stops" with the departure/arrival status set to "added". See the [disruption](#disruption) objects section for its full content and description.
+
+<div></div>
+### Journeys
+
+``` shell
+# Request example for /journeys
+http://api.navitia.io/v1/coverage/<coverage>/journeys?from=<origin>&to=<destination>&data_freshness=realtime
+```
+
+``` shell
+# Extract of the public transport section of the response /journeys
+
+    "sections": ⊖[
+       ⊖{
+            "additional_informations": ⊕[1 item],
+            "arrival_date_time": "20190605T204500",
+            "base_arrival_date_time": "20190605T204500",
+            "co2_emission": ⊕{2 items},
+            "data_freshness": "realtime",
+            "departure_date_time": "20190605T160000",
+            "display_informations": ⊕{13 items},
```

Comment:
Same: it's probably worth expanding display_information.links here (and add a word in description)

Reply:
TO DO by @pbougue